### DISCUSSION TITLE
Can I ask you why `bun` does not have `nullzero` by default unlike `go-pg`?

### DISCUSSION THREAD

**Discussion by frederikhors** [uptrace/bun#498] [Q&A] [answered]
Hi @vmihailenco, can I ask you why `bun` does not have `nullzero` by default unlike `go-pg`?

I'm using and asking for Postgres.

Is there any difference in storing empty strings, 0 or NULL in PG?

**Comment by vmihailenco** [comment]
It is disabled by default because most/some users don't expect this behavior.

**Comment by frederikhors** [comment, marked as answer]
Reading here and there online (eg https://stackoverflow.com/questions/1222314/does-setting-not-null-on-a-column-in-postgresql-increase-performance) I think it's best to avoid NULL values when possibile.

I will revert each `nullzero`.
